Unable to build on linux - mismatched types

My build is failing with the latest version of web-view (`0.6.2`) with the error:

```
error[E0308]: mismatched types
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/webview-sys-0.5.0/gtk.rs:132:13
    |
132 |             b"\0".as_ptr() as *const i8
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u8, found i8
    |
    = note: expected type `*const u8`
               found type `*const i8`
```

`gtk.rs` was modified 20 days ago (https://github.com/Boscop/web-view/commit/44ed909ea3defb29d8d786e25d5bb26d379ee8c3). I am not sure if I miss something in my build or if the aforementioned commit is the culprit. Any ideas?

--- a/webview-sys/gtk.rs
+++ b/webview-sys/gtk.rs
@@ -129,7 +129,7 @@ unsafe extern "C" fn webview_new(
     webkit_web_view_load_uri(
         mem::transmute(webview),
         if url.is_null() {
-            b"\0".as_ptr() as *const i8
+            b"\0".as_ptr() as *const _
         } else {
             url
         },

